rotkehlchen/globaldb/handler.py

```diff
@@ -1048,3 +1048,51 @@ def get_historical_price_data(source: HistoricalPriceOracle) -> List[Dict[str, A
              'from_timestamp': entry[2],
              'to_timestamp': entry[3],
              } for entry in query]
+
+    @staticmethod
+    def rebuild_assets_list() -> Tuple[bool, str]:
+        """
+        Delete current information about assets and repopulate with the last
+        builtin version
+        """
+        root_dir = Path(__file__).resolve().parent.parent
+        builtin_database = root_dir / 'data' / 'global.db'
+        remove_assets = 'DELETE FROM assets;'
+        remove_tokens = 'DELETE FROM ethereum_tokens;'
+        atach_datbase = f'ATTACH DATABASE "{builtin_database}" AS clean_db;'
+        check_version = 'SELECT value from clean_db.settings WHERE name=="version";'
+        insert_assets = 'INSERT INTO assets SELECT * FROM clean_db.assets;'
```

Comment:
I really don't see why you need to add them as extra strings here and call them down there. You don't seem to be reusing any of them apart from the detach database. Just write them out normally. Makes the function appear a lot bigger than needed.